presto-main/src/main/java/io/prestosql/server/InternalAuthenticationManager.java

```diff
@@ -68,13 +72,13 @@ public InternalAuthenticationManager(Optional<String> sharedSecret, String nodeI
 
     public boolean isInternalRequest(HttpServletRequest request)
     {
-        return request.getHeader(PRESTO_INTERNAL_BEARER) != null;
+        return internalJwtEnabled && request.getHeader(PRESTO_INTERNAL_BEARER) != null;
```

Comment:
is this required. When `internalJwtEnabled` is `false` then `filterRequest` won't add `PRESTO_INTERNAL_BEARER`

Reply:
This is not needed.